### PR TITLE
use percy instead of storyshots

### PR DESCRIPTION
**Pull request by tkidman**
## What it does, and why

> Add a thorough explanation of what the code in this PR does in this section.

## Checklist

> Please go through following checklist before requesting review and fix issues listed there. If your code touches the files with the below in question, please address them.

#### Workflow

- [ ] Are you writing documentation/comments for the bits of code with complex changes?
- [ ] Are you writing tests (Storybook/Unit testing)

## Jira Tickets

> If this PR implements changes related to one or more Jira tickets please add them here:

## UI Changes

> If this PR introduce some interaction or animation change please also add GIF with how it behaves. Don't forget that this change should be possible to simulate in storybook.
> You can use [this tool](http://gifbrewery.com/) to record your screen and convert it to a GIF.

### CODE REVIEW

File: .github/workflows/percy.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1.0.0
+      - name: Install dependencies
+        run: npm i
+      - name: Percy Test
+        working-directory: packages/orcs-design-system
+        uses: percy/storybook-action@v0.1.2
+        with:
+          percy-flags: "--widths=320,1024"
```
Comment (abottega) [inline .github/workflows/percy.yml:27]: We should remove 320 and 2014 for now if we are just going to be testing 1280.

File: packages/orcs-design-system/package.json [hunk 2]
```diff
@@ -35,7 +35,8 @@
     "deploy-client": "serverless client deploy  --no-confirm",
     "storybook": "start-storybook -s ./lib/assets",
     "build-storybook": "build-storybook -s .storybook/static",
-    "deploy-storybook": "storybook-to-ghpages"
+    "deploy-storybook": "storybook-to-ghpages",
+    "snapshot": "build-storybook && percy-storybook --widths=320,1280"
```
Comment (abottega) [inline packages/orcs-design-system/package.json:39]: Also remove 320 from here.

**Review by abottega — COMMENTED** [review]
Left two comments otherwise all good

File: .github/workflows/percy.yml (hunk 1 above)
Comment (tkidman) [reply .github/workflows/percy.yml]: done

**Pull request merged**